resources/js/processes-catalogue/components/CardProcess.vue

```diff
@@ -95,6 +95,13 @@ export default {
      * Build URL for Process Cards
      */
     buildURL() {
+      if (this.category === undefined || this.category.id === -1) {
+        return "processes?status=ACTIVE"
```

Comment:
Please use this API
`process_bookmarks/processes`,

Reply:
@pmPaulis that API only return the bookmarked and not all processes

Reply:
This api works like the processes